app/helpers/application_helper.rb

```diff
@@ -1,9 +1,14 @@
 module ApplicationHelper
+
+  def current_demo_night?
+    DemoNight.current != nil
+  end
+
   def accepting_submissions?
     DemoNight.current != nil && DemoNight.current.accepting_submissions?
   end
 
-  def current_demo_night?
-    DemoNight.current != nil
+  def voting?
+    DemoNight.current != nil && DemoNight.current.voting?
```

Comment:
can we do `current_demo_night? && DemoNight.current.voting?`? Just to keep it dry